README.md

```diff
@@ -81,17 +88,57 @@ of Chimney. It drops you immediately into a REPL session.
 ```
 curl -s https://raw.githubusercontent.com/scalalandio/chimney/master/try-chimney.sh | bash
 Loading...
-Welcome to the Ammonite Repl 1.1.0
-(Scala 2.12.4 Java 1.8.0_152)
-If you like Ammonite, please support our development at www.patreon.com/lihaoyi
-@ case class Foo(x: String, y: Int) 
+Welcome to the Ammonite Repl 2.2.0 (Scala 2.13.10 Java 17.0.1)
```

Comment:
Ammonite example section is duplicated, maybe it's enough to have just one?

Reply:
The code is duplicated but not how you call it. E.g. in our script we import things for users in scala-cli we have to show them how to add library to Ammonite REPL (we can do it via command argument, but we need to `import io.scalaland.chimney._` in REPL).

Alternatively, we might just leave only Scala CLI section, since it's going to be a standard tool for kicking off and it embeds ammonite already.

Reply:
I'm all for having just one example how to quickly get started. Scala CLI sounds good 👍 

Would you mind modifying the docs section? It's also available there: https://scalalandio.github.io/chimney/getting-started/quickstart.html#using-ammonite